jest-dynamodb-config.js

```diff
@@ -12,8 +12,10 @@ module.exports = async () => {
     port,
     clientConfig: {
       endpoint,
-      sslEnabled: false,
-      region: "local",
+      credentials: {
```

Comment:
In my testing, I got prompts to fetch new AWS SSO credentials unless I provided dummy credentials for the local DynamoDB.